tests/integration/security/sds_ingress/ingress_test.go

```diff
@@ -48,7 +48,7 @@ func TestMain(m *testing.M) {
 func TestSingleTlsGateway_SecretRotation(t *testing.T) {
 	framework.
 		NewTest(t).
-		Features("security.ingress.tls").
+		Features("security.ingress.tls.secretrotation").
```

Comment:
sometime you have `-` like `server-keyroration` and sometimes you don't, can we be consistent?

Reply:
make both of them consistent to secretrotation